keras_nlp/models/albert/albert_tokenizer_test.py

```diff
@@ -52,17 +41,15 @@ def test_tokenizer_basics(self):
         )
 
     def test_errors_missing_special_tokens(self):
-        bytes_io = io.BytesIO()
-        sentencepiece.SentencePieceTrainer.train(
-            sentence_iterator=iter(["abc"]),
-            model_writer=bytes_io,
-            vocab_size=5,
-            pad_id=-1,
-            eos_id=-1,
-            bos_id=-1,
-        )
         with self.assertRaises(ValueError):
-            AlbertTokenizer(proto=bytes_io.getvalue())
+            AlbertTokenizer(
+                proto=str(
+                    pathlib.Path(__file__).parent.parent.parent
+                    / "tests"
+                    / "test_data"
+                    / "sentencepiece_bad.proto"
```

Comment:
Maybe let's be more specific than `"bad"` here.

`"no_special_token_vocab.spm"`

Reply:
Done.